packages/api/tests/lambdas/test-manual-consumer.js

```diff
@@ -303,13 +297,7 @@ test.serial('handler sets envs from event', async (t) => {
 test.serial('handler should not overwrite existing envs', async (t) => {
```

Comment:
```suggestion
test.serial('handler should not overwrite existing timestamp envs', async (t) => {
```

Reply:
Arguably these two tests can now just directly test `configureTimestampEnvs` but that's probably out of scope.